engine/src/main/java/com/datatorrent/stram/plan/physical/StreamMapping.java

```diff
@@ -222,6 +289,12 @@ private void redoMapping() {
       plan.undeployOpers.addAll(currentUnifiers);
       addSlidingUnifiers();
 
+      List<PTOutput> unifierSources = this.upstream;
+      Partitioner<Operator> partitioner = streamMeta.getSource().getUnifierMeta().getAttributes().contains(Context.OperatorContext.PARTITIONER) ?
```

Comment:
If ```streamMeta.getSource().getUnifierMeta().getAttributes().contains(Context.OperatorContext.PARTITIONER)``` and ```(Partitioner<Operator>)streamMeta.getSource().getUnifierMeta().getValue(Context.OperatorContext.PARTITIONER)``` were extracted to variables this would be easier to read and understand what's going on